flink-connectors/flink-connector-kafka/src/main/java/org/apache/flink/connector/kafka/sink/KafkaCommitter.java

```diff
@@ -112,6 +112,8 @@ public void commit(Collection<CommitRequest<KafkaCommittable>> requests)
                         e);
                 recyclable.ifPresent(Recyclable::close);
```

Comment:
What about removing this line as the `recyclable` will finally be closed anyway?